quickwit/quickwit-config/resources/tests/config/quickwit.json

```diff
@@ -1,70 +1,71 @@
 # Comments are supported.
 {
```

Comment:
What's changed?

Reply:
NVm, I guess you added a `merge_concurrency` property, but somehow the formatting changed.

Reply:
I'll fix the indentation.

Reply:
It's fine.